.isort.cfg

```
@@ -0,0 +1,14 @@
+# https://github.com/timothycrosley/isort/wiki/isort-Settings
+[settings]
+default_section = THIRDPARTY
+include_trailing_comma = true
+indent = 4
+known_first_party = towncrier
+line_length = 79
+lines_after_imports = 2
+# https://github.com/timothycrosley/isort#multi-line-output-modes
+multi_line_output = 5
+no_lines_before = LOCALFOLDER
+sections=STDLIB,TESTING,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
```

Maybe make it a bit more sparse?
```suggestion
sections = STDLIB, TESTING, FRAMEWORKS, THIRDPARTY, FIRSTPARTY, LOCALFOLDER
```